packages/@n8n/api-types/src/schemas/booleanLiteral.ts

```diff
@@ -0,0 +1,3 @@
+import { z } from 'zod';
+
+export const booleanLiteral = z.enum(['true', 'false']).transform((value) => value === 'true');
```

Comment:
now that we are transforming this to actual boolean, should it still be called `booleanLiteral`, or should it rename it to `booleanFromString`?

We can also change the transform predicate to use a `.toLowerCase`, just in case.

Reply:
> We can also change the transform predicate to use a .toLowerCase, just in case.

I can do that, but then I have to change `z.enum` to `z.string` and then it will turn any string that is not a variation of `"true"` to `true`. That means `"invalid"` instead of returning an error.

Reply:
true. let's leave it as it is then.